ProfileView/src/index.ts

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// This file imports all of the webcomponents from "components" folder
+
+import {
+  html,
+  internalProperty,
+  property,
+  customElement,
+  LitElement
+} from "lit-element";
+// import "./components/ActivityStream/ActivityItem";
+// import "./components/View/View";
+import "@cjaas/common-components";
+import { Profile } from "./types/cjaas";
+import styles from "./assets/styles/View.scss";
+
+/**
+ * Please give your widget a unique name. We recommend using prefix to identify the author and help avoid naming conflict. e.g. "2ring-timer-widget"
+ */
+@customElement("cjaas-profile-view-widget")
+export default class CjaasProfileWidget extends LitElement {
+  @property() customer: string | undefined;
```

Comment:
In general, we need to describe props we expect in README

Reply:
Did those not appear? I did add some descriptions